src/Numerics/ODESolvers/AdditiveRungeKuttaMethod.jl

```diff
@@ -790,19 +865,116 @@ function ARK2GiraldoKellyConstantinescu(
         RT(1 / (2 * sqrt(2))) RT(1 / (2 * sqrt(2))) RT(1 - 1 / sqrt(2))
     ]
 
-    RKB = [RT(1 / (2 * sqrt(2))), RT(1 / (2 * sqrt(2))), RT(1 - 1 / sqrt(2))]
-    RKC = [RT(0), RT(2 - sqrt(2)), RT(1)]
+    RKB_explicit =
+        [RT(1 / (2 * sqrt(2))), RT(1 / (2 * sqrt(2))), RT(1 - 1 / sqrt(2))]
+    RKC_explicit = [RT(0), RT(2 - sqrt(2)), RT(1)]
+    # For this ARK method, both RK methods share the same
+    # B and C vectors in the Butcher table
+    RKB_implicit = RKB_explicit
+    RKC_implicit = RKC_explicit
 
-    Nstages = length(RKB)
+    Nstages = length(RKB_explicit)
 
     AdditiveRungeKutta(
         F,
         L,
         backward_euler_solver,
         RKA_explicit,
         RKA_implicit,
-        RKB,
-        RKC,
+        RKB_explicit,
+        RKB_implicit,
+        RKC_explicit,
+        RKC_implicit,
+        split_explicit_implicit,
+        variant,
+        Q;
+        dt = dt,
+        t0 = t0,
+        nsubsteps = nsubsteps,
+    )
+end
+
+"""
+    Trap2LockWoodWeller(F, L, backward_euler_solver, Q; dt, t0, nsubsteps,
+                        split_explicit_implicit, variant)
+
+This function returns an [`AdditiveRungeKutta`](@ref) time stepping object,
+see the documentation of [`AdditiveRungeKutta`](@ref) for arguments definitions.
+This time stepping object is intended to be passed to the `solve!` command.
+
+The time integrator scheme used is Trap2(2,3,2) with δ_s = 1, δ_f = 0, from
+the following reference
+
+### References
+    @article{Ascher1997,
+      title = {Numerical analyses of Runge–Kutta implicit–explicit schemes
+               for horizontally explicit, vertically implicit solutions of
+               atmospheric models},
+      author = {S.-J. Lock and N. Wood and H. Weller},
+      volume = {140},
+      number = {682},
+      pages = {1654-1669},
+      year = {2014},
+      journal = {Quarterly Journal of the Royal Meteorological Society},
+      publisher = {{RMetS}}
+    }
+"""
+function Trap2LockWoodWeller(
+    F,
+    L,
+    backward_euler_solver,
+    Q::AT;
+    dt = nothing,
+    t0 = 0,
+    nsubsteps = [],
+    split_explicit_implicit = false,
+    variant = NaiveVariant(),
+    δ_s = 1,
+    δ_f = 0,
+    α = 0,
+) where {AT <: AbstractArray}
+
+    @assert dt !== nothing
+    # In this scheme B and C vectors do not coincide,
+    # hence we can't use the LowStorageVariant optimization
+    @assert variant isa NaiveVariant
+
+    T = eltype(Q)
+    RT = real(T)
+
+    #! format: off
+    RKA_explicit = [
+        RT(0)    RT(0)    RT(0)    RT(0)
+        RT(δ_s)  RT(0)    RT(0)    RT(0)
+        RT(1/2)  RT(1/2)  RT(0)    RT(0)
+        RT(1/2)  RT(0)    RT(1/2)  RT(0)
+    ]
+
+    RKA_implicit = [
+        RT(0)                  RT(0)                  RT(0)    RT(0)
+        RT(δ_f * (1 - α) / 2)  RT(δ_f * (1 + α) / 2)  RT(0)    RT(0)
+        RT(1/2)                RT(0)                  RT(1/2)  RT(0)
+        RT(1/2)                RT(0)                  RT(0)    RT(1/2)
+    ]
```

Comment:
Super pedantic, but can we put spaces between the numbers and division (to match the formatter?)

Also, you could use rational `1 // 2` before casting to floating point type.

Reply:
Thanks! I will add the space, but I'd rather follow the convention used in the other ARK methods, where I didn't see using the rational division ` // `. I would keep a single `/` for consistency.